### ISSUE TITLE
amazonaws.com/ClusterClient/PHP-7.0/latest-64bit -- Unable to load

### ISSUE THREAD

**Issue by Kspen** [awslabs/aws-elasticache-cluster-client-memcached-for-php#17]
Hello,

I'm revisiting this project after over a year.  I have been hosting a site on AWS for about 2 years, hoping one day to use Elasticache.  Previously, I ended up rolling my own session caching server but willing to pay the difference and switch to Elasticache.

When trying to run the precompiled memcached client from https://elasticache-downloads.s3.amazonaws.com/ClusterClient/PHP-7.0/latest-64bit , PHP 7.0.26 says:

PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/amazon-elasticache-cluster-client.so' - /usr/local/lib/php/extensions/no-debug-non-zts-20151012/amazon-elasticache-cluster-client.so: undefined symbol: executor_globals in Unknown on line 0

Another "oddity" is that your README.markdown says PHP 7.x is not supported but other documentation within AWS explicitly offers a PHP 7.0 EC Cluster Client binary.

Seriously considering moving to GCP at this point.

**Comment by QuChen88** [comment]
Can you please provide more information on your set up? Namely the AMI you are running on and how you set up PHP7? That can help us reproduce this problem

**Comment by Kspen** [comment]
The AMI is Debian 8.  PHP compiled using the following config (copied from config.nice):

'./configure' \
'--with-apxs2=/usr/local/apache2/bin/apxs' \
'--with-zlib' \
'--enable-sockets' \
'--disable-ipv6' \
'--with-mcrypt' \
'--enable-mbstring' \
'--with-jpeg-dir' \
'--with-kerberos' \
'--with-curl' \
'--enable-zip' \
'--with-openssl' \
'--enable-soap' \
'--with-xmlrpc' \
'--with-pdo-mysql' \
'--with-bz2' \
'--with-gd' \
'--with-freetype-dir' \
'--enable-gd-native-ttf' \
'--with-gettext' \
'--enable-opcache' \
'--with-xsl' \
'--enable-intl' \

Standard install.  Let me know if you need info on any other software version information.

Thanks for the quick response.

**Comment by QuChen88** [comment]
I was able to install our precompiled PHP7.0 memcached client artifact on Debian 8 (Jessie) AMI. Here is the set of steps I did:

1. Launch an EC2 instance from Debian 8 Jessie AMI
2. Install PHP 7.0 via standard installation. I didn't compile PHP 7.0 myself manually, but instead followed the set of steps here https://shiji.info/blog/2016/en/99
3. Downloaded our pre-compiled php7.0 memcached client from the link you posted, and run "tar -xvf" to extract the artifact "amazon-elasticache-cluster-client.so", copied it under the directory /usr/lib/php/20151012, and finally updated /etc/php/7.0/cli/php.ini file to include the extension
4. When I launch PHP7 in interactive mode, I noticed that the library is loaded successfully. i.e. 

$ php -a
Interactive mode enabled
php > $m = new Memcached();
